web/src/compose_setup.ts

```diff
@@ -133,11 +144,12 @@ export function initialize() {
         (event) => {
             event.preventDefault();
             const {$banner_container, is_edit_input} = get_input_info(event);
+            assert(event.target instanceof HTMLElement);
```

Comment:
Is declaring the type for `event` correctly a better way to do this? `git grep 'instanceof HTMLEl'` has plenty of examples, so if so, we should just file an issue for sweeping that.

Reply:
the type for `event` is already implied as `ClickEvent`, and that doesn't specify `event.target` clearly enough. So unless you're suggesting something I haven't thought of, I don't think that would work.